server/controllers/stock/core.js

```diff
@@ -99,9 +99,12 @@ function getLotFilters(parameters) {
   filters.equals('lot_uuid', 'lot_uuid', 'm');
   filters.equals('inventory_uuid', 'uuid', 'i');
   filters.equals('consumable', 'consumable', 'i');
+  filters.equals('is_asset', 'is_asset', 'i');
   filters.equals('group_uuid', 'uuid', 'ig');
   filters.equals('text', 'text', 'i');
   filters.equals('label', 'label', 'l');
+  filters.equals('is_asset', 'is_asset', 'i');
```

Comment:
It looks like you have `is_asset` here listed twice.

Reply:
Done